react-components/src/components/RevealTopbar/RuleBasedOutputsContainer.tsx

```diff
@@ -0,0 +1,100 @@
+/*!
+ * Copyright 2024 Cognite AS
+ */
+import { useState, type ReactElement, useEffect, useMemo } from 'react';
+
+import { Button, Dropdown, Flex, Icon, Menu, Switch } from '@cognite/cogs.js';
+
+import { RuleBasedOutputs } from '../RuleBasedOutputs/RuleBasedOutputs';
+import { RULE_BASED_OUTPUTS_VIEW } from '../RuleBasedOutputs/constants';
+import { FdmSDK } from '../../utilities/FdmSDK';
+import { useSDK } from '../RevealCanvas/SDKProvider';
+import { type RuleAndEnabled } from '../RuleBasedOutputs/types';
+
+export const RuleBasedOutputsContainer = (): ReactElement => {
+  const [ruleInstances, setRuleInstances] = useState<RuleAndEnabled[]>();
+  const [currentRuleSetEnabled, setCurrentRuleSetEnabled] = useState<any>();
+  const sdk = useSDK();
+  const fdmSdk = useMemo(() => new FdmSDK(sdk), [sdk]);
+
+  useEffect(() => {
+    // eslint-disable-next-line @typescript-eslint/explicit-function-return-type
+    const getAllInstances = async () => {
+      const versionedPropertiesKey = `${RULE_BASED_OUTPUTS_VIEW.externalId}/${RULE_BASED_OUTPUTS_VIEW.version}`;
+
+      const filter = {
+        in: {
+          property: [RULE_BASED_OUTPUTS_VIEW.space, versionedPropertiesKey, 'shamefulOutputTypes'],
+          values: ['color']
+        }
+      };
+      const result = await fdmSdk.filterAllInstances(filter, 'node', RULE_BASED_OUTPUTS_VIEW);
+
+      const rulesAndEnabled: RuleAndEnabled[] = [];
+      result.instances.forEach((instance) => {
+        rulesAndEnabled.push({
+          isEnabled: false,
+          rule: instance
+        });
+      });
+
+      setRuleInstances(rulesAndEnabled);
+    };
+
+    void getAllInstances();
+  }, []);
+
+  const onChange = (data: { target: { id: any; checked: boolean } }): void => {
+    ruleInstances?.forEach((item) => {
+      item.isEnabled = false;
+    });
+    const selectedRule = ruleInstances?.find((item) => {
+      return item.rule.properties.id === data.target.id && data.target.checked;
+    });
+
+    setRuleInstances(ruleInstances);
+    setCurrentRuleSetEnabled(selectedRule);
+
+    if (selectedRule === undefined) return;
+    selectedRule.isEnabled = data.target.checked;
+  };
+
+  return (
+    <>
+      <Dropdown
+        placement="top-start"
+        className="rule-based-outputs-class"
+        content={
+          <Menu
+            style={{
+              maxHeight: 300,
+              overflow: 'auto',
+              marginBottom: '20px'
+            }}>
+            <Menu.Header>Select The RuleSet</Menu.Header>
```

Comment:
This needs to use translation - can you ask @pramodcog What the translation key will be? Then you use it by writing `const { t } = useTranslation();` above, inside this component, and then `{t(<translation-key>)}` here

Reply:
added here. 
https://github.com/cognitedata/reveal/pull/4210/commits/d748037cc48301eb04ef215bd184c86e4158a622

@pramodcog will add the translation item from another PR